doc/theory/glossary.tex

```diff
@@ -308,72 +313,63 @@
 	\sup_{x \in D_1 \cup \Delta} ||x - \gamma(x)||_{\infty.}
 	\end{equation*}
 	
+	The set of persistence diagrams together with any of the distances above is a \hyperref[metric_space]{metric space}.
```

Comment:
Here, I forgot to implement one of the changes I asked: this portion should be

> The set of persistence diagrams together with any of the distances above is a
% \hyperref[metric_space]{metric space}
metric space.